### PR TITLE
dependency-updates

### PR DESCRIPTION
**Pull request by cbardini**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,18 +19,18 @@
       <dependency> 
         <groupId> com.github.osisoft.ocs_sample_library_preview</groupId>
         <artifactId>ocs_sample_library_preview</artifactId>
-        <version>[0.1.5-preview,)</version>
+        <version>[0.1.6-preview,)</version>
```
Comment (kelseyduffy) [inline pom.xml:22]: last wait a couple days for the library dependency update to merge, then bump this to 0.1.7

Comment (cbardini) [reply pom.xml]: Updated!

**Review by anton-dahlin — APPROVED** [review]

**Review by derekendres — APPROVED** [review]

**Review by kelseyduffy — APPROVED** [review]

**Comment by kelseyduffy** [comment]
looks like this needs the 0.1.9 version of the java library in order to build. The PR behind this one for ADH branding bumps it to 0.2, which also works. That PR looks to also bump the jupiter API versions which are the whole point of this PR, so I approved it in case we want to merge it and move on, but it might be better to just close this PR entirely and merge the ADH branding one behind it?

**Pull request closed without merging**